bin/pd/Makefile

```diff
@@ -33,6 +33,7 @@ ckanapi := $(REGISTRY_CKANAPI_COMMAND)
 ogc_search := $(OGC_SEARCH_COMMAND)
 oc_search := $(OC_SEARCH_COMMAND)
 backup_dir := $(PD_BACKUP_DIRECTORY)
+tmp_dir := $(PD_TMP_DIRECTORY)
```

Comment:
```suggestion
tmp_dir := $(or $PD_TMP_DIRECTORY,$TMPDIR)
```

Reply:
Hmm. this doesn't work exactly because $TMPDIR can be unset then we'll get an error on mktemp

Reply:
Updated suggestion, this one should work in all cases and not require changes in our dev environments